pypeit/spectrographs/gemini_gmos.py

```diff
@@ -133,6 +134,35 @@ def default_pypeit_par():
 
         return par
 
+    def config_specific_par(self, scifile, inp_par=None):
+        """
+        Modify the PypeIt parameters to hard-wired values used for
+        specific instrument configurations.
+
+        .. todo::
+            Document the changes made!
+
+        Args:
+            scifile (str):
+                File to use when determining the configuration and how
+                to adjust the input parameters.
+            inp_par (:class:`pypeit.par.parset.ParSet`, optional):
+                Parameter set used for the full run of PypeIt.  If None,
+                use :func:`default_pypeit_par`.
+
+        Returns:
+            :class:`pypeit.par.parset.ParSet`: The PypeIt paramter set
+            adjusted for configuration specific parameter values.
+        """
+        par = self.__class__.default_pypeit_par() if inp_par is None else inp_par
+
+        headarr = self.get_headarr(scifile)
+
+        # Turn PCA off for long slits
```

Comment:
Note this hack of mine for longslit.  I think it is ok..

Reply:
I worry a bit about this, but as long as 'arcsec' always picks out the longslit data, I think this is fine...